src/lib/store.js

```diff
@@ -42,7 +42,9 @@ const initialState = {
 
   // When a user lands on the page, check if they have accepted our
   // cookie policy.
-  userAcceptsCookies: false,
+  // We disable this check in dev and test environments so the cookie banner
+  // doesn't interfere with testing.
+  userAcceptsCookies: config.prod ? false : true,
```

Comment:
super big nit but I think this might be more readable as

```js
   userAcceptsCookies: !config.prod,
```

Reply:
and update the comment to have fewer negatives, so e.g. "We automatically accept cookies anywhere but prod so that ..."